content/docs/1.10.0/advanced-resources/rebuilding/offline-replica-rebuilding.md

```diff
@@ -29,6 +29,21 @@ Starting with v1.9.0, Longhorn supports offline replica rebuilding, allowing deg
 - When triggered, Longhorn attaches the volume without activating the frontend, rebuilds any missing replicas, and then detaches the volume upon completion.
 - This process can be interrupted if the workload scales up.
 
+## Resource Constraints
```

Comment:
> ## Resource Constraints

The ticket is a bit unclear. When I read this section, I can't tell what message it’s supposed to deliver. Could you come up with a more suitable title?